### ISSUE TITLE
iOs : Works on browser, emulator, but not on device with XCode

### ISSUE THREAD

**Issue by crouti** [thgreasi/localForage-cordovaSQLiteDriver#6]
Anyone succeded in making it work on ios devices ?
It just never enters the  defineDriver callback, with no error

```
localforage.defineDriver(window.cordovaSQLiteDriver).then(function() {
                console.log("Setting localForage drivers...");
                return localforage.setDriver([
                    // Try setting cordovaSQLiteDriver if available,
                    window.cordovaSQLiteDriver._driver,
                    // otherwise use one of the default localforage drivers as a fallback.
                    // This should allow you to transparently do your tests in a browser
                    localforage.INDEXEDDB,
                    localforage.WEBSQL,
                    localforage.LOCALSTORAGE
                ]);
            }).then(function() {
                // this should alert "cordovaSQLiteDriver" when in an emulator or a device    
                console.log("Driver type = "+localforage.driver());
                alert(localforage.driver());
            });
```

Any idea ?
Thank you very much

**Comment by thgreasi** [comment]
Never tested it on an iOS device.
Is `window.cordovaSQLiteDriver` actually defined?
I will try to reach a colleague for some feedback on this.

**Comment by cory-buckley** [comment]
I think this might have something to do with the Content Security Policy. I couldn't successfully recall data without making sure I had `script-src *;` in my CSP meta tag. Also, you may need the following in your config.xml
`<allow-intent href="*"/>`
`<allow-navigation href="*"/>`

**Comment by racekarl** [comment]
This driver works as expected for me using the cordova-sqlite-storage driver version 0.7.14, but not using the latest 1.2.0 version of the plugin.  I believe the issue is that the cordova-sqlite-storage driver introduced a breaking change where the database location must be passed when calling openDatabase.  It does not appear that this driver does so.

**Comment by thgreasi** [comment]
This should be fixed in #7. I couldn't find enough time during last
weekend, I hope the coming one will be more productive for me. Can you give
it a try and provide some feedback whether it solves the issue?

> ## 
> 
> _Thodoris Greasidis_
> Computer, Networking &
> Communications Engineer

**Comment by thgreasi** [comment]
Should this be closed now that #7 got finally merged?
Do you still face the same issues?

**Comment by crouti** [comment]
i didn't, but i stopped using localForage, so i close it now
